pyuntl/quality.py

```diff
@@ -64,12 +64,12 @@ def determine_completeness(py_untl):
                     else:
                         completeness_dict['%s' % i.tag]['present'] = True
     # Get total score of the pyuntl object.
-    for k, v in completeness_dict.iteritems():
+    for k, v in completeness_dict.items():
         # If presence was toggled true, adjust score based on weight.
         if v['present']:
             py_untl_object_score += completeness_dict[k]['weight']
     # Calculate the float score completeness.
-    completeness = py_untl_object_score / total_points
+    completeness = py_untl_object_score // total_points
```

Comment:
Either the comment is wrong or this conversion is wrong. If one of those operands is a float, then the result in Python 2 would have been a float, in which case `/` would be correct. If they are both integers, then `//` is correct, but then the comment needs to change.

Reply:
Hmm...you seem to be right. I missed the comment and that `py_untl_object_score = 0.0` is declared to be a float. It should be a normal division.